etc/testing/travis.sh

```diff
@@ -68,7 +68,7 @@ function test_bucket {
 
     echo "Running bucket $bucket_num of $num_buckets"
     # shellcheck disable=SC2207
-    tests=( $(go test -v  "${package}" -list ".*" | grep -v ok | grep -v Benchmark) )
+    tests=( $(go test -v  "${package}" -list ".*" | grep -v '^ok' | grep -v '^Benchmark') )
```

Comment:
This was filtering any test with `ok` in it, like say "Token". Looks like it affected 14/89 tests in the auth package.